package.json

```diff
@@ -78,28 +78,28 @@
   },
   "homepage": "https://github.com/bbc/simorgh#readme",
   "dependencies": {
-    "@bbc/gel-foundations": "3.1.0",
+    "@bbc/gel-foundations": "3.2.0",
     "@bbc/psammead-assets": "^1.1.1",
-    "@bbc/psammead-brand": "^4.2.0",
-    "@bbc/psammead-caption": "^2.1.0",
-    "@bbc/psammead-consent-banner": "^2.1.0",
-    "@bbc/psammead-copyright": "^1.1.0",
+    "@bbc/psammead-brand": "^4.2.3",
+    "@bbc/psammead-caption": "^2.1.2",
+    "@bbc/psammead-consent-banner": "^2.1.3",
+    "@bbc/psammead-copyright": "^1.1.2",
     "@bbc/psammead-figure": "^1.1.1",
-    "@bbc/psammead-headings": "^3.0.1",
-    "@bbc/psammead-image": "^1.1.0",
-    "@bbc/psammead-image-placeholder": "^1.1.0",
-    "@bbc/psammead-inline-link": "^1.1.10",
-    "@bbc/psammead-locales": "^1.1.0",
-    "@bbc/psammead-media-indicator": "^2.3.1",
+    "@bbc/psammead-headings": "^3.0.2",
+    "@bbc/psammead-image": "^1.1.2",
+    "@bbc/psammead-image-placeholder": "^1.1.2",
+    "@bbc/psammead-inline-link": "^1.2.2",
+    "@bbc/psammead-locales": "^1.2.1",
+    "@bbc/psammead-media-indicator": "^2.3.2",
     "@bbc/psammead-navigation": "^2.1.0",
-    "@bbc/psammead-paragraph": "^2.1.0",
-    "@bbc/psammead-section-label": "^2.1.0",
-    "@bbc/psammead-sitewide-links": "^3.0.0",
-    "@bbc/psammead-story-promo": "^2.3.0",
-    "@bbc/psammead-story-promo-list": "^1.1.0",
-    "@bbc/psammead-styles": "^1.2.0",
+    "@bbc/psammead-paragraph": "^2.1.2",
+    "@bbc/psammead-section-label": "^2.1.3",
+    "@bbc/psammead-sitewide-links": "^3.0.1",
+    "@bbc/psammead-story-promo": "^2.3.1",
+    "@bbc/psammead-story-promo-list": "^1.1.2",
+    "@bbc/psammead-styles": "^2.0.1",
     "@bbc/psammead-timestamp-container": "^2.3.2",
-    "@bbc/psammead-visually-hidden-text": "^1.1.0",
+    "@bbc/psammead-visually-hidden-text": "^1.1.1",
```

Comment:
Why are all these version numbers being updated? I thought all the bumping had been taken care of separately.

Reply:
In psammead but not in Simorgh , if not updated we will have duplicates issue, 
we need to use updated packages that have psammead-styles @2.0.1 as a  dependency which has the new Nassim fonts